react/ProductListWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { SizeMe } from 'react-sizeme'
+
+interface Props {
+  items: Item[]
+  onQuantityChange: (uniqueId: string, value: number) => void
+  onRemove: (uniqueId: string) => void
+}
+
+const ProductListWrapper: StorefrontFunctionComponent<Props> = props => {
+  return (
+    <SizeMe>
+      {({ size }) =>
+        size.width && size.width < 640 ? (
```

Comment:
Maybe move it to a constant with a meaningful name, e.g.: `MAX_MOBILE_WIDTH`.

Reply:
Agreed. I thought of doing it when I was testing this `SizeMe` component but then I forgot 🤦‍♂